src/System.Private.ServiceModel/src/System/ServiceModel/Channels/BufferedWriteStream.cs

```diff
@@ -34,21 +33,25 @@ namespace System.ServiceModel.Channels
     /// </summary>
     internal sealed class BufferedWriteStream : Stream
     {
-        public const int DefaultBufferSize = 4096;
+        public const int DefaultBufferSize = 8192;
```

Comment:
>8192 [](start = 45, length = 4)

Non-blocking question only: why this change?  Perf improvement?  Is there any embedded dependency on this being 8K?